scripts/docker/Dockerfile

```diff
@@ -10,54 +10,54 @@ FROM golang:${VERSION} AS builder
 ARG CGO_ENABLED=0
 ARG BUILD_TAGS
 
-WORKDIR /go/src/github.com/hashicorp/vault
+WORKDIR /go/src/github.com/openbao/openbao
 COPY . .
 
 RUN make bootstrap \
-  && CGO_ENABLED=$CGO_ENABLED BUILD_TAGS="${BUILD_TAGS}" VAULT_DEV_BUILD=1 sh -c "'./scripts/build.sh'"
+  && CGO_ENABLED=$CGO_ENABLED BUILD_TAGS="${BUILD_TAGS}" OPENBAO_DEV_BUILD=1 sh -c "'./scripts/build.sh'"
 
 # Docker Image
 
 FROM alpine:3.13
 
-# Create a vault user and group first so the IDs get set the same way,
+# Create a bao user and group first so the IDs get set the same way,
 # even as the rest of this may change over time.
-RUN addgroup vault && \
-    adduser -S -G vault vault
+RUN addgroup bao && \
```

Comment:
Not to nitpick, (sorry!), but I think we've previously agreed in https://github.com/openbao/openbao/pull/44#discussion_r1457955470 that we could retain `openbao` in directory/path names (and presumably, users/groups as well) for clarity and differentiation.

`bao` as a CLI though is shorter to type than OpenBao, and so we preferred `bao` for just that.

Reply:
Environment variables are good to keep as `BAO_` though, it doesn't seem like anyone had issues with that: https://github.com/openbao/openbao/issues/66

Reply:
done :)